k8s/client.go

```diff
@@ -821,3 +821,64 @@ func (client *Client) GetClusterInfo() (ClusterInfo, error) {
 	info.Platform = serverVersion.Platform
 	return info, nil
 }
+
+func (client *Client) DeleteCoreInstance(ctx context.Context, name, environment string, shouldWait bool) error {
```

Comment:
yes, this deletes everything we create when we are instantiating sync.

Reply:
actually this is removing way more than what should be installed with sync 
It looks like you are removing the operator

Reply:
just look like, because we create serviceaccount, cluster role bindings, etc etc for the sync also: https://github.com/calyptia/cli/blob/main/cmd/coreinstance/create_core_instance_operator.go#L140-L210

alo the naming isnt helping but after we delete the other k8s command and pass to use just this one it will improve

Reply:
Why not use the same permissions as granted to the operator?